roles/docker_latest_setup/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+# vim: set ft=ansible:
+#
+# This role configures the atomic host to run docker lates
+#
+
```

Comment:
There should be a check at the top of this role to see if `docker-latest` is installed.

I just found that F25AH doesn't not include the package, so the playbook bombed out.